app/custom/templates/blog-single/index.js

```diff
@@ -0,0 +1,3 @@
+/* Add requires here for all directives referenced by the home template */
+require( './../../organisms/header/directive' );
```

Comment:
No reason, I can update.

However, @andybar2 is this necessary? Should patternlab automatically include them in the bundle?